js/sketch.js

```diff
@@ -85,8 +87,20 @@ let seedInventory = {
   milk: 5,
 };
 
+let cookedInventory = {
```

Comment:
going to leave these as 1 as part of development because cooking is taking a while. made a note to change before final submission